### ISSUE TITLE
Can we display a label for slider with steps?

### ISSUE THREAD

**Issue by praswebdev** [darul75/ng-slider#75]
Hi,
Can we display a label for the slider with steps, which moves with the pointer?

**Comment by darul75** [comment]
good idea, I see what I can do with dimension string, maybe with a specific character to handle prefix or suffix.

**Comment by praswebdev** [comment]
Thanks. Also can we get a break or an indicator on the scale for each step

**Comment by olegakbarov** [comment]
That would be really useful, i'm stumbled upon the need of it.

**Comment by darul75** [comment]
need time but ok I will stop what I was doing to implement it first ;)

**Comment by olegakbarov** [comment]
i'm actually WIP with this stuff and came up with this monkey-patch:

At first i added an array of label values to options object
```javascript
var OPTIONS = {
...
discreteValues: ["value1", "value2", "value3"]
...
}
```
than in directive i've rewritten the defaults 
```javascript
var OPTIONS = {
from: 0,
to: scope.options.discreteValues.length - 1
...
discreteValues: scope.options.discreteValues
}
```
and finally just set the value to map to corresponding string in discreteValues array  
```javascript
this.o.labels[pointer.uid].value.html(
this.settings.discreteValues[this.nice(pointer.value.origin)]
);
```
let me know what you think, i can send a PR if you think this is fine solution

**Comment by darul75** [comment]
`dimension` is given ability to set suffix string, would a prefix string enough for you ?

`dimension` "> km" or " km" : will display "1 km", "2 km"
`dimension` "< km"  : will display "km 1", "km 1"

I was thinking about that...

**Comment by darul75** [comment]
hmm or just display scale values for pointer when pointers on it ?

**Comment by olegakbarov** [comment]
Thanks for reply!

Suffix string would not be option for me. I need to display strings without numbers :/
I need a value that works like dropdown:
![Dropdown-like slider](http://cl.ly/image/0T1s2U3f0k2I/)

I think binding a model to this value would be useful case. Kinda dropdown-ish.

**Comment by darul75** [comment]
ok, maybe still not closed enough to what you mean but can you check this one:

http://jsfiddle.net/darul75/3kt5mv8w/1/

value binding is set in options with json obj "keyModel:valueLable"

modelLabels: {1: 'WTF', 2: 'OK'},

bit it is display on top of pointers ?

**Comment by olegakbarov** [comment]
Yeah, this fiddle looks like what i need! So the question is how to attach modelLabels to the top fo pointer?

**Comment by darul75** [comment]
to the bottom you mean ? ;)

**Comment by olegakbarov** [comment]
Hahaha, yeah! I guess i need one more cup of coffee.

**Comment by darul75** [comment]
I have to go now but check that asap
